connector_prestashop_catalog_manager/__openerp__.py

```diff
@@ -3,17 +3,19 @@
 # Copyright 2011-2013 Akretion
 # Copyright 2015 AvanzOSC
 # Copyright 2015-2016 Tecnativa
+# Copyright 2018 Meta IT
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Prestashop-Odoo Catalog Manager",
-    "version": "8.0.1.0.2",
+    "version": "8.0.1.0.3",
     "license": "AGPL-3",
     "depends": [
         "connector_prestashop"
     ],
     "author": "Akretion,"
               "AvanzOSC,"
+              "Meta IT,"
```

Comment:
For this little contribution you shouldn't be in author list, but as contributor in the README